ansible_collections/arista/cvp/molecule/cv_container_v3/converge.yml

```diff
@@ -0,0 +1,42 @@
+---
```

Comment:
Can we follow the same format as the other molecule tests?
move these tests into a separate playbook and import it in converge.yml?

Reply:
Fixed!